src/adapter/search-response-adapter/total-hits-adapter.ts

```diff
@@ -0,0 +1,20 @@
+import type { MeiliSearchResponse } from '../../types'
+
+export function adaptTotalHits(
+  searchResponse: MeiliSearchResponse<Record<string, any>>
+): number {
+  const {
+    hitsPerPage = 0,
+    totalPages = 0,
+    estimatedTotalHits = undefined,
+    totalHits = undefined,
+  } = searchResponse
+  if (estimatedTotalHits !== undefined) {
```

Comment:
Do you think this `estimatedTotalHits` can be null? Because if it is `null` we will have a `null` return.

Reply:
As far as all the typed SDK's, it cannot be null